edx_repo_tools/conventional_commits/commitstats.py

```diff
@@ -0,0 +1,197 @@
+"""
+Collect and report on conventional commit statistics.
+"""
+
+import contextlib
+import datetime
+import fnmatch
+import os
+import os.path
+import re
+import sqlite3
+import subprocess
+import sys
+
+import click
+try:
+    import dataset
+    import matplotlib.pyplot as plt
+    import matplotlib.dates
+    import pandas as pd
+except ImportError as err:
+    sys.exit(f"Did you install requirements/conventional_commits.txt? {err}")
+
+
+
+@click.group(help=__doc__)
+def main():
+    pass
+
+def get_cmd_output(cmd):
+    """Run a command in shell, and return the Unicode output."""
+    try:
+        data = subprocess.check_output(cmd, shell=True, stderr=subprocess.STDOUT)
+    except subprocess.CalledProcessError as ex:
+        data = ex.output
+    try:
+        data = data.decode("utf-8")
+    except UnicodeDecodeError:
+        data = data.decode("latin1")
+    return data
+
+def load_commits(db, repo_name):
+    """Load the commits from the current directory repo."""
+
+    SEP = "-=:=-=:=-=:=-=:=-=:=-=:=-=:=-"
+    GITLOG = f"git log --no-merges --format='format:date: %aI%nhash: %H%nauth: %aE%nname: %aN%nsubj: %s%n%b%n{SEP}'"
+    SHORT_LINES = 5
+
+    # $ git log --format="format:---------------------%ndate: %aI%nhash: %H%nauth: %aE%nname: %aN%nsubj: %s%n%b"
```

Comment:
Looks like you changed the output format, can you update to show a sample of the new format or delete this comment?

Reply:
done.